extensions/search-router/package.json

```diff
@@ -63,17 +66,15 @@
     "@types/react": "18.3.3",
     "eslint": "^8.57.0",
     "prettier": "^3.3.3",
-    "sqlite": "^5.1.1",
-    "sqlite3": "^5.1.7",
     "typescript": "^5.4.5"
   },
   "scripts": {
     "build": "ray build",
     "dev": "ray develop",
     "fix-lint": "ray lint --fix",
     "lint": "ray lint",
+    "download-kagi-bangs": "npx ts-node scripts/download-kagi-bangs.ts && npm run fix-lint",
```

Comment:
Document this command in the readme for future development?

Reply:
thanks, updated in https://github.com/raycast/extensions/pull/17996/commits/8bf27afd23482205ea17c2efc76c3c6c0ea23129